docs/source/quickstart.mdx

```diff
@@ -0,0 +1,389 @@
+
+Hello! This tutorial gets you up and running with Apollo Federation. Unlike other articles in this documentation, this article uses some Apollo tools and terminology that are still in **preview**, and therefore this content is not yet recommended for use in a production environment.
+
+We expect these tools to leave preview soon, so this tutorial is a great way to begin learning about federation using upcoming best practices.
+
+> **This tutorial requires Node.js version 12 or later.** 
+
+## Federation concepts
+
+In a federated architecture, multiple GraphQL APIs are **composed** into a single federated data graph. The individual APIs are called **subgraphs**, and they're composed into a **supergraph**:
+
+```mermaid
+graph BT;
+  gateway(["Supergraph (A + B + C)"]);
+  serviceA[Subgraph A];
+  serviceB[Subgraph B];
+  serviceC[Subgraph C];
+  gateway --- serviceA & serviceB & serviceC;
+```
+
+Usually, each subgraph corresponds to a different service in your backend. The supergraph is then represented by a **gateway**, which routes each incoming query to the appropriate combination of subgraphs and returns the combined result.
+
+The supergraph's schema is the combination of each subgraph's schema, plus some special federation-specific directives. This architecture enables clients to query data from multiple subgraphs simultaneously, just by querying the supergraph.
+
+Now that we have a high-level understanding of federation concepts, let's jump in.
+
+## 1. Install the Rover CLI
+
+**Rover** is Apollo's new CLI for managing data graphs, including federated ones. It's currently in preview.
+
+<p><a href="https://www.apollographql.com/docs/rover/getting-started/#installation-methods" target="_blank">Install Rover</a></p>
+
+After installing, run `rover` in your terminal with no arguments to confirm that it installed successfully. We'll use various Rover commands in later steps.
+
+## 2. Create a gateway
+
+> You can also [clone our example gateway on GitHub](https://github.com/apollographql/supergraph-demo). The GitHub example is **not** identical to the gateway you create in this tutorial, but it demonstrates the same concepts.
+
+As mentioned in the [preface](#preface-federated-architecture), your federated supergraph is represented by a **gateway** that routes queries to various subgraphs. For this tutorial, we'll use some Apollo-hosted example services as our subgraphs, and we'll set up our own gateway in front of them.
+
+With the help of the `@apollo/gateway` library, [Apollo Server](https://www.apollographql.com/docs/apollo-server/) can act as our federated gateway.
+
+In a new Node.js project, first install the required libraries:
+
+```
+npm install apollo-server @apollo/gateway
```

Comment:
```suggestion
npm install apollo-server @apollo/gateway @apollo/rover
```

Reply:
Gonna keep installing Rover as its first standalone step but updating to specifically instruct to `npm install -g` instead of punting to Rover docs